draft-ietf-rtcweb-security-arch.xml

```diff
@@ -637,91 +637,78 @@ Syntax:
+         </t>
+         <section title="Generating the Initial SDP Offer" anchor="sec.sdp-id-attr-oa-inio">
+         <t>
+           When an offerer sends an offer, in order to provide its identity
+           assertion to the peer, it includes an 'identity' attribute in the
+           offer. In addition, it MUST include SDP 'fingerprint' attributes for
+           each fingerprint that has been used to assert the identity.
+         </t>
+         </section>
+         <section title="Answerer Processing of SDP Offer" anchor="sec.sdp-id-attr-oa-anso">
+         <t>
+           When an answerer receives an offer that contains an 'identity'
+           attribute, the answerer can use the the attribute information to
+           contact the IdP, and verify the identity of the peer.  If the
+           answerer does not accept the attribute value, or if the identity
+           verification fails, it MUST reject the offer.
+         </t>
+         </section>
+         <section title="Generating of SDP Answer" anchor="sec.sdp-id-attr-oa-ansa">
+         <t>
+           When an answerer sends an answer, in order to provide its identity
+           assertion to the peer, it includes an 'identity' attribute in the
+           answer. In addition, it MUST include 'fingerprint' attributes for
+           each fingerprint that have been used to assert the identity.
+         </t>
+         <t>
+           NOTE: The answerer can include an 'identity' attribute in the answer
+           independently of whether the corresponding offer contained an
+           identity attribute or not.
+         </t>
+         </section>
+         <section title="Offerer Processing of SDP Answer" anchor="sec.sdp-id-attr-oa-offa">
+         <t>           
+           When the offerer receives an answer that contains an 'identity'
+           attribute, the offerer can use the attribute information to contact
+           the IdP, and verify the identity of the peer. If the offerer does
+           not accept the attribute (or the attribute value), or if the identity
+           verification fails, it MUST take proper actions to reject the answer.
+         </t>
+         </section>
+         <section title="Modifying Session" anchor="sec.sdp-id-attr-oa-modi">
+         <t>
+           When an endpoint has provided its identity assertion in a previous
+           successful offer/answer exchange, and when the endpoint later in the 
+           session sends an offer or answer, the endpoint can either keep the 
+           identity assertion value unchanged, modify the value, or revoke the 
+           identity assertion. This section describes the procedures for doing so.
+         </t>
+         <t>
+           If the endpoint wants to maintain the previously provided identity assertion, 
+           it MUST include an identity attribute with the previously provided attribute value
+           in the offer or answer.
```

Comment:
Do you want to say something here about needing to include the fingerprints ?

Reply:
Yes, I will add text about that.